### DISCUSSION TITLE
Linux和Windows系统下相同代码的差异

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#154] [Q&A] [answered]
我使用了相同的代码（除中英文），但是linux系统下第一次输入一定是无效的，这让我很费解，也不知道如何上网查询这个bug，希望各位能给予一点帮助
![image](https://user-images.githubusercontent.com/100075314/160232805-5150c0de-3c3e-41c3-b69e-6c308f63400f.png)
![image](https://user-images.githubusercontent.com/100075314/160232849-b2c0fc85-5423-4ea1-8fa3-0ca2fdf202f2.png)

**Comment by VansamaZWT** [comment]
我尚不会在Linux中设置断点，这让我的问题难以解决

**Reply by binbin00** [reply]
碰到这种很具体的代码bug，你可以直接和你的责任助教沟通，甚至让ta帮你debug.

**Comment by RicoloveFeng** [comment, marked as answer]
对于不方便断点的情况，你可以保存读取到的东西，然后逐字符（甚至是字符对应的数字）打印出来，看看到底接收了什么东西。
